sdk/src/wallet/account/mod.rs

```diff
@@ -296,6 +297,15 @@ impl AccountInner {
         Ok(all_addresses.to_vec())
     }
 
+    /// Returns the first address of the account as bech32
+    pub async fn first_address_bech32(&self) -> Option<Bech32Address> {
```

Comment:
Not sure how I feel about this tbh, sounds way too specific

Reply:
its gonna be 90% of use cases for users though. sounds nice to have

Reply:
Should probably be removed with the single address wallet :trollface: